src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -253,21 +253,32 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the scope map.', required=True)
 
+    valid_actions = "Valid actions are metadata/read, metadata/write, content/read, content/write and content/delete"
+    with self.argument_context('acr scope-map update') as c:
+        c.argument('add_repository', options_list=['--add'], nargs='+', action='append', required=False,
+                   help='repository permissions to be added. Use the format "--add REPO [ACTION1 ACTION2 ...]" per flag. ' + valid_actions)
+        c.argument('remove_repository', options_list=['--remove'], nargs='+', action='append', required=False,
+                   help='respsitory permissions to be removed. Use the format "--remove REPO [ACTION1 ACTION2 ...]" per flag. ' + valid_actions)
+
     with self.argument_context('acr scope-map create') as c:
-        c.argument('add_repository', options_list=['--add'], nargs='+', help='Actions to be added. Use the format "--add REPO [ACTION1 ACTION2 ...]" per flag. Valid actions are metadata/read, metadata/write, content/read, content/write and content/delete', action='append', required=True)
+        c.argument('repositories', nargs='+', action='append', required=True,
+                   help='repository permissions. Use the format "--repositories REPO [ACTION1 ACTION2 ...]" per flag. ' + valid_actions)
 
     with self.argument_context('acr token') as c:
         c.argument('registry_name', options_list=['--registry', '-r'])
         c.argument('token_name', options_list=['--name', '-n'], help='The name of the token.', required=True)
         c.argument('scope_map_name', options_list=['--scope-map'], help='The name of the scope map associated with the token', required=False)
-        c.argument('status', options_list=['--status'], help='The status of the token. Allowed values are "enabled" or "disabled".', required=False, default="enabled")
+        c.argument('status', options_list=['--status'], arg_type=get_enum_type(['enabled', 'disabled']),
+                   help='The status of the token', required=False, default="enabled")
 
     with self.argument_context('acr token create') as c:
-        c.argument('scope_map_name', options_list=['--scope-map'], help='The name of the scope map associated with the token', required=True)
+        c.argument('scope_map_name', options_list=['--scope-map'],
+                   help='The name of the scope map with pre-configured repository permissions. Use "--repositories" if you would like CLI to configure one for you')
+        c.argument('repositories', nargs='+', action='append',
+                   help='repository permissions. Use the format "--repositories REPO [ACTION1 ACTION2 ...]" per flag. ' + valid_actions)
```

Comment:
I am changing it to be `--repository` like you recommended which will be consistent with others

Reply:
@yugangw-msft 
My comment was about the argument variable `c.argument('repositories'` not the command line argument.  I suggested changing it to a more descriptive variable such as `c.argument('repository_actions_list'` since it's a list of Repositories and their corresponding actions.

I should've been clearer in the review. It's a nit suggestion. Feel free to change it to a more descriptive variable if possible.